new-lamassu-admin/src/pages/Notifications/sections/FiatBalanceAlerts.js

```diff
@@ -13,27 +14,41 @@ import styles from './FiatBalanceAlerts.styles.js'
 
 const useStyles = makeStyles(styles)
 
-const schema = Yup.object().shape({
-  fiatBalanceCassette1: Yup.number()
-    .integer()
-    .min(0)
-    .required(),
-  fiatBalanceCassette2: Yup.number()
-    .integer()
-    .min(0)
-    .required()
-})
-
 const NAME = 'fiatBalanceAlerts'
 
-const FiatBalance = ({ section }) => {
+const FiatBalance = ({
+  section,
+  max = Number.MAX_SAFE_INTEGER,
+  fieldWidth = 80
+}) => {
   const { isEditing, isDisabled, setEditing, data, save } = useContext(
     NotificationsCtx
   )
   const classes = useStyles()
 
   const editing = isEditing(NAME)
 
+  const schema = Yup.object().shape({
+    fiatBalanceCassette1: Yup.number()
+      .integer()
+      .min(0)
+      .max(max)
+      .required(),
+    fiatBalanceCassette2: Yup.number()
+      .integer()
+      .min(0)
+      .max(max)
+      .required()
+  })
+
+  const fiatBalanceCassette1Percent = data?.fiatBalanceCassette1
+    ? (100 * data?.fiatBalanceCassette1) / max
+    : 0
```

Comment:
Probably better to do 

` 100 * (data?.fiatBalanceCassette1 ?? 0) / max`

Instead of the ternary if, same applies for the second cassette